src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -45,14 +45,14 @@ public AddCommand parse(String args) throws ParseException {
         }
 
         Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
```

Comment:
I'm wondering if `name` should be changed to something like `title` instead to be even clearer that it doesn't refer to the seller's name?

Reply:
To make this as foolproof as possible, I will change ```name``` to ```propertyName``` for edit/addCommand.
These are the places I am unable to do OOP on ```Person``` since ```Person``` is immutable but data has to be mutable.